schemas/when_rule/compare_to_value.json

```diff
@@ -15,6 +15,7 @@
         "equals",
         "not equals",
         "contains",
+        "contains any",
```

Comment:
Not sure why this is needed here?

It is already defined in: https://github.com/ONSdigital/eq-questionnaire-validator/blob/master/schemas/when_rule/compare_to_multiple_values.json#L5-L15

Reply:
If you look at the other PR on this story, its falling over because it is missing, but I will take a look.

Reply:
https://github.com/ONSdigital/eq-questionnaire-schemas/pull/28/checks?check_run_id=535935251

Reply:
Yes, but I am confused to why it breaks when it is defined as:
https://github.com/ONSdigital/eq-questionnaire-validator/blob/master/schemas/when_rule/definitions.json#L7-L23

Reply:
`contains any` needs to be used with `values` rather than `value`